Source/Settings.as

```diff
@@ -110,6 +110,9 @@ bool Setting_Gamepad_SteerPercentage = false;
 [Setting category="Gamepad" name="Steer percentage size" drag min=2 max=40]
 int Setting_Gamepad_SteerPercentageSize = 16;
 
+[Setting category="Gamepad" name="Text color" color]
```

Comment:
Can we make this say `Text and symbol color` instead? To tell the user that the symbols are also affected by this.

Reply:
Yeah, good idea